app/next-client-app/app/(protected)/scanreports/[id]/layout.tsx

```diff
@@ -45,6 +45,15 @@ export default async function ScanReportLayout({
     { name: "Downloads", slug: "downloads", iconName: "Download" },
   ];
 
+   {/* Pushed the Edit button to the Items above so it appears on same level */}
+  if (canEdit) {
+    items.push({
+      name: "Edit Scan Report",
```

Comment:
```suggestion
      name: "Edit Details",
```
To be consistent with the Dataset page

Reply:
@AndyRae Sure. Fixed